docker-compose.yaml

```diff
@@ -374,6 +374,12 @@ services:
       - ./scripts/docker-compose/imports/smartdocuments-wiremock/mappings:/home/wiremock/mappings
     entrypoint: [ "/docker-entrypoint.sh", "--global-response-templating", "--disable-gzip", "--verbose" ]
 
+  office-converter:
```

Comment:
Goed om ook even toe te voegen aan het lijstje services in de `installDockerCompose.md`.